pkg/rancher-desktop/components/SnapshotCard.vue

```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+import Vue from 'vue';
+
+import { Snapshot } from '@pkg/main/snapshots/types';
+
+import type { PropType } from 'vue';
+
+function formatDate(value: string) {
+  if (!value) {
+    return 'n/a';
+  }
+
+  const leadingZero = (v: number) => `${ v < 10 ? '0' : '' }${ v }`;
+  const d = new Date(value);
+
+  return `${ d.getFullYear() }-${ leadingZero(d.getMonth() + 1) }-${ leadingZero(d.getDate()) } ${ leadingZero(d.getHours()) }:${ leadingZero(d.getMinutes()) }`;
+}
+
+export default Vue.extend<any, any, any, any>({
```

Comment:
Try to avoid the usage of `any` wherever possible. We should be able to make use of just `Vue.extend` if we provide a type annotation for the `snapshot` computed prop.

```suggestion
export default Vue.extend({
```